src/resources/mathjax_preview.js

```diff
@@ -36,6 +36,47 @@ var isEmptyMathJax = function(text) {
     return text.replace(/\$/g, '').trim().length == 0;
 };
 
+// MathJax v3 compat (http://docs.mathjax.org/en/latest/upgrading/v2.html#v2-compatibility-example)
```

Comment:
You just modify the preview stuff. MathJax is also used in the read mode, like `markdown-it.js` and `markdown-template.js`. Do we need to change code in those places?

Reply:
You're right. I added this snippet to them.